product_packaging_template_sale/models/product_packaging_template.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductPackagingTemplate(models.Model):
+    _inherit = "product.packaging.template"
+
+    sales = fields.Boolean(
+        default=True, help="If true, the packaging can be used for sales orders"
+    )
+
+    def write(self, vals):
+        """
+        Propagate changes on packages at variant level.
+        """
+        res = super().write(vals)
+        if "sales" in vals:
+            self.mapped("packaging_ids").write({"sales": vals["sales"]})
```

Comment:
Any reason to not override the `_get_value_to_propagate` method?

 https://github.com/OCA/product-attribute/pull/2133/files#diff-e72ab97e2c338a6b34d6056d6e046d8a57a672c21d7a7f295c28d8239d21354fR48

Reply:
No, the `_get_value_to_propagate` method was added afterwards, due to a review on the other PR, and I forgot to change it here. Done now.